flowdb/sql/04_schema_other.sql

```diff
@@ -68,3 +68,15 @@ Schema used for temp storage during etl.
 */
 
 CREATE SCHEMA IF NOT EXISTS etl;
+
+CREATE TYPE cdrtype AS ENUM ('voice', 'sms', 'mds');
+CREATE TYPE status AS ENUM ('in_process','done', 'quarantine');
```

Comment:
Nitpick (missing space):
```suggestion
CREATE TYPE status AS ENUM ('in_process', 'done', 'quarantine');
```